packages/create-neutrino-project/commands/init/templates/taskcluster-web-library/src/index.js

```diff
@@ -0,0 +1 @@
+export { sum, product } from './my-library';
```

Comment:
We don't need the extra import and export dance here. Let's export directly from `src/index.js` our functionality:

```js
export default () => console.log(/* Maybe a "Welcome to <directory name>" here.*/);
```